internal/notifier/forwarder.go

```diff
@@ -40,7 +43,10 @@ func NewForwarder(hookURL string, proxyURL string) (*Forwarder, error) {
 }
 
 func (f *Forwarder) Post(event recorder.Event) error {
-	err := postMessage(f.URL, f.ProxyURL, event)
+	err := postMessage(f.URL, f.ProxyURL, event, func(req *retryablehttp.Request) {
+		req.Header.Set("Notification-Controller", event.ReportingController)
```

Comment:
Please extract the header name into a variable and rename it to `gotk-component`